src/web/src/pages/opportunities/[[...query]]/index.tsx

```diff
@@ -64,28 +63,42 @@ export const getStaticProps: GetStaticProps = async () => {
     organizations: null,
     zltoRewardRanges: null,
   });
-  const opportunities_latestCourses = await searchOpportunities({
+  const opportunities_learning = await searchOpportunities({
```

Comment:
the "types" property should be used to filter the opportunities by Learning/Task. According to Adrian, the ID of the types should be consistent across environments, so we can hard-code the type ids in the constants.ts file for now.